mongo_connector/oplog_manager.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, primary_client, doc_managers,
         self.continue_on_error = kwargs.get('continue_on_error', False)
 
         # Set of fields to export
-        self._fields = set(kwargs.get('fields', []))
+        self.fields = set(kwargs.get('fields', []))
```

Comment:
I think this should be self._fields, is there a reason why you're setting self.fields instead?